lib/src/icons.dart

```diff
@@ -63,6 +64,42 @@ class GYWIcon {
       );
 }
 
+/// Converts a [GYWIcon] object to and from JSON.
+class GYWIconJsonConverter
+    implements JsonConverter<GYWIcon, Map<String, dynamic>> {
+  /// Constructor.
+  const GYWIconJsonConverter();
+
+  @override
+  GYWIcon fromJson(Map<String, dynamic> json) {
```

Comment:
Can you add the methods `fromJson` and `toJson` in a flutter test to ensure they are coherent and that we can make `fromJson(font.toJson())` and inversely ?